package.json

```diff
@@ -68,5 +68,8 @@
     "yargs": "^10.0.3",
     "zalgo-promise": "^1.0.40",
     "zoid": "^6.0.74"
+  },
+  "dependencies": {
+    "smoothscroll-polyfill": "^0.4.4"
```

Comment:
Ik heb nog eens nagedacht over mijn eerdere comment dat dit een dependency moest zijn, maar ik wist toen niet dat het niet over een app, maar over een component ging. In dit laatste geval had je toch gelijk dat dit een `devDependency` moet zijn.

Reply:
done